docs/pages/by-operation.md

```
@@ -0,0 +1,117 @@
+</script>
+
+<OAOperation :spec="spec" :operationId="operationId" :isDark="isDark" />
+```
+
+</template>
+
+</ScopeConfigurationTabs>
+
+## Searching Operations
+
+If you want to make use of search on your site, the default `local` search will not work due to https://github.com/vuejs/vitepress/issues/2939
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)